### DISCUSSION TITLE
Other hardware while a new official is being researched

### DISCUSSION THREAD

**Discussion by iamk3** [WLAN-Pi/feedback#10] [General Inquiries] [answered]
Hi all!

I have been lookin into the WLAN-Pi for a while and really like the project. I was hoping to finally purchase one of my own along with the case that was designed. Since the former kit is no longer in production, is there a temporary hardware solution and/or guide for another board? I have several RPi laying around.

Thanks again for the work that has been put into this project!

**Comment by joshschmelzle** [comment, marked as answer]
@iamk3 if you can wait until the end of the month we will have a new official released.

**Reply by iamk3** [reply]
I can do that! Thank you!

**Comment by joshschmelzle** [comment]
Hey @iamk3 we've launched our new hardware and also have an arm64 image that works on RBPi 3b+ and RBPi 4 units.

New hardware images are live on our website: https://www.wlanpi.com/

And the latest image can be found here: https://github.com/WLAN-Pi/pi-gen/releases/latest

**Reply by dotty37** [reply]
Hi, I would love to put this together, or get a box already build.
How do we do that? where do we get the hardware ?

**Reply by jiribrejcha** [reply]
The WLAN Pros store should eventually have the WLAN Pi Pro in stock: https://wlanprofessionals.com/store/

Alternatively, Badger Wi-Fi in Europe:
https://www.badgerwifi.co.uk/

As Josh mentioned, we are also exploring a WLAN Pi Community Edition, which will have limited feature, lower price tag, and will be based on off-the-shelf hardware.

If you have a RPi4, it will boot our WLAN Pi Pro image, and you need to add a Wi-Fi adapter that supports monitor mode. For example https://m.aliexpress.com/item/4000048659616.html?gatewayAdapt=Pc2Msite
